### ISSUE TITLE
Trying to deploy CPU pod : `Error: required flag(s) "gpuType" not set`

### ISSUE THREAD

**Issue by gwpl** [runpod/runpodctl#161]
Dear Maintainers,

I am trying to deploy CPU pod (for testing),

and as much as tool seems working with GPU pods, for CPU pods I get:

`Error: required flag(s) "gpuType" not set`

Extra user wish: additionally, probably providing command lines examples in WebUI or documentation would be also helpful for convenient parameter settings or to have templates to tweak or follow.

Thank you for great service.

**Comment by matanyall** [comment]
Yeah, we need cpu serverless support here if we are going to be able to use this feature of runpod

**Comment by gwpl** [comment]
But I didn't want to spin on serverless... I wanted to spin up instances, parform on them CPU intense tasks, and bring them down.

**Comment by TimPietruskyRunPod** [comment]
Resolved in [v2.0.0](https://github.com/runpod/runpodctl/releases/tag/v2.0.0).

```bash
runpodctl pod create \
  --template-id <template-id> \
  --compute-type CPU \
  --volume-in-gb 20 \
  --name my-cpu-pod
```

CPU pods are now supported via `--compute-type CPU`, which skips the GPU requirement.
